private-templates-service/adaptivecards-templating-service/index.ts

```diff
@@ -0,0 +1,9 @@
+import { TemplateServiceClient } from '../adaptivecards-templating-service/src/TemplateServiceClient'
```

Comment:
We can remove this file. Grace has made docs on how to make the import in `server/src/app.ts` work over here: https://github.com/microsoft/adaptivecards-templates/tree/dev/private-templates-service/adaptivecards-templating-service#developer-instructions